### PR TITLE
Fix issue plazi/ebiodiv/98

### PR DESCRIPTION
**Pull request by dalf**
image

Close https://github.com/plazi/eBioDiv/issues/98 :
> user guidance to what the user has to do

### CODE REVIEW

File: src/components/CurationList.vue [hunk 1]
```diff
@@ -49,12 +48,20 @@
                 </td>
                 <td colspan="4"></td>
```
Comment (emilie19) [inline src/components/CurationList.vue:49]: remove  <td colspan="4"></td>

**Review by emilie19 — COMMENTED** [review]
décalage dans le tableau de l'entité selectionnée lorsqu'on clique sur le expand

File: src/components/CurationList.vue [hunk 2]
```diff
@@ -49,12 +48,20 @@
...
+                    <br /><br />
+                    <div class="separator">
+                        <h2><span>{{ get_curation_name }}s associated with the {{ get_occurrence_name.toLowerCase() }} {{ occurrences_selection.key }}</span></h2>
+                    </div>
+                    <p>
+                        {{ processed_curation.length }} suggested 
+                        {{ get_curation_name.toLowerCase() }}{{ processed_curation.length > 1?'s':'' }}
+                        to curate
+                    </p>
+                    <p class="notice">Please indicate for each suggested material citation whether it matches the specimen or not (Yes /No).</p>
```
Comment (dalf) [inline src/components/CurationList.vue:62]: Add a space after `Yes /`

File: src/components/CurationList.vue [hunk 3]
```diff
@@ -1,18 +1,18 @@
 <template>
 
     <div class="component-container" :style="cssVars">
-
-        <div class="separator">
-            <h2><span>{{ get_curation_name }}s associated with the {{ get_occurrence_name.toLowerCase() }} {{
```
Comment (emilie19) [inline src/components/CurationList.vue:6]: inversion between curation_name and occurrence_name

File: src/components/CurationList.vue (hunk 1 above)
Comment (dalf) [reply src/components/CurationList.vue]: Done

File: src/components/CurationList.vue (hunk 2 above)
Comment (dalf) [reply src/components/CurationList.vue]: Done

**Comment by dalf** [comment]
Last change in 586689d : `{{ occurrences_selection.verbatimLabel }}` removed since it is also displayed in the details.

**Pull request merged**